File: client-sdk/csharp/sdk/src/VertexCache/Sdk/Protocol/Parsers/OkResponseParser.cs
using VertexCache.Sdk.Results;

namespace VertexCache.Sdk.Protocol.Parsers
{
    public class OkResponseParser : IResponseParser
    {
        public bool CanParse(string response) => response == "+OK";

        public VCacheResult Parse(string response) => VCacheResult.SuccessWithRaw(response);
    }
}

File: client-sdk/go/sdk/protocol/parsers/ok_response_parser.go
package parsers

import "vertexcache/sdk/results"

type OkResponseParser struct{}

func (p *OkResponseParser) CanParse(response string) bool {
	return response == "+OK"
}

func (p *OkResponseParser) Parse(response string) *results.VCacheResult {
	return results.NewSuccessWithRaw(response)
}
